__tests__/generate-from-template.spec.js

```diff
@@ -60,40 +63,43 @@ describe('generateFromTemplate', () => {
     jest.spyOn(Store.prototype, 'get').mockImplementation(getMock);
     jest.spyOn(Store.prototype, 'set').mockImplementation(setMock);
   });
-  it('should call the generatorBanner, and all 5 steps', async () => {
+  it('should call the generatorBanner, and all 6 steps', async () => {
     await generateFromTemplate({ templateName: 'ejs@1.0.0' });
 
-    expect(log.goToStep).toHaveBeenCalledTimes(5);
+    expect(log.goToStep).toHaveBeenCalledTimes(6);
     expect(log.goToStep).toHaveBeenNthCalledWith(1, 1);
```

Comment:
@JAdshead @code-forger Found a really neat tool https://github.com/atlassian/jest-in-case add test variations without too much boiler plate, maybe we could use it here? 

Here is an example testing a similar cli tool like ours. (the tests are not great but you get the idea) https://github.com/kentcdodds/kcd-scripts/blob/7bc29e41e46e73b4b57c0f975648a90a75c24c80/src/scripts/__tests__/lint.js

Reply:
it is similar to `test.each` but the api looks clearer 🤔

https://jestjs.io/docs/api#testeachtablename-fn-timeout

Reply:
might save on boilerplate but I think it makes it less clear whats being tested.